bpf/lib/common.h

```diff
@@ -220,13 +221,21 @@ struct lb4_service {
 	__u16 port;
 	__u16 count;
 	__u16 rev_nat_index;
+	__u16 weight;
 } __attribute__((packed));
 
 struct lb4_reverse_nat {
 	__be32 address;
 	__u16 port;
 } __attribute__((packed));
 
+/* Must match lbmap.go maxSeq */
+#define LB_RR_MAX_SEQ 31
+struct lb_sequence {
+	__u16 count;
+	__u16 idx[LB_RR_MAX_SEQ];
```

Comment:
Hmm, this config option applies to go side as well which is more tricky to do it at compile time. I am trying to see if go supports compile time constants. So far the only thing I found is go run -ldflags="-X main.var foo". I do not know if its a neat way to set constants in go. Much more simpler on the bpf side. Let me see if I can come up with something cleaner on the go side.